pt/lessons/libraries/distillery.md

```diff
@@ -0,0 +1,246 @@
+---
+version: 2.0.1
+title: Distillery (Básico)
+---
+
+Distillery é um gerenciador de releases escrito em Elixir puro. Ele permite que você produza releases que podem ser instaladas em outros lugares com pouca ou nenhuma configuração.
```

Comment:
```suggestion
Distillery é um gerenciador de releases escrito em Elixir puro. Ele permite que você produza releases que podem ser rodadas em outros lugares com pouca ou nenhuma configuração.
```
I don't know if `instaladas` is the best translation for `deployed` is this specific case. In my opinion, here `instaladas` doesn't fully translate the meaning of the word. @elixirschool/portuguese Do you any suggestion on how we can better translate this?

Reply:
Yeah i agree. I also thought about using `distribuidas` but that doens't seem to to fit well either.

IMO i would maintain the original word as well. The issue as i see it are the conjugations, like `deployar`, `deployadas`, and so on. But those still sounds better than the other translations.

Just let me know and ill change to whatever you guys think its best. :)

Reply:
I would keep the original word in this case, `deployed`, as Thiago pointed out.

Reply:
@anderkonzen just to be sure, the current phrase is like this:
`... que podem ser **rodadas** em outros lugares...`

do i change it to:
`... que podem ser **deployed** em outros lugares...`

or : 
`... que podem ser **deployadas** em outros lugares...`
?

Reply:
I think `... que podem ser deployed em outros lugares ...`. Still looks a bit weird to me but seems correct at the moment.

I also saw this word is used in the "Custom commands" section, we can adjust there as well.

Reply:
Done. :)
Let me know if anything else comes up.

Reply:
Nice :)